pop-api/integration-tests/src/fungibles/mod.rs

```diff
@@ -281,7 +281,7 @@ fn decrease_allowance_works() {
 		assets::start_destroy(&addr, token);
 		assert_eq!(
 			decrease_allowance(&addr, token, &BOB, amount),
-			Err(Module { index: 52, error: [16, 0] }),
+			Err(Module { index: 52, error: [10, 0] }),
```

Comment:
What is changing here?

Reply:
Because we check the `allowance` first before `cancel_approval` called, this returns `Unapproved` instead of `AssetNotLive`. (This should be handled in the `decrease_allowance` PR but I missed that, sorry for this mess)

Reply:
But then the test need to be changed. We are destroying the asset one live above to trigger the error `NotLive`